docs/guides/telemetryservice/pushtoTDEngine.md

```diff
@@ -15,19 +15,27 @@ spec:
     SQLSetting:
       serverAddress: 192.168.14.163:6041
       username: root
-      secret: taosdata
+      secret: my-secret
```

Comment:
need to update to include username injection

Reply:
ok